daprdocs/content/en/developing-applications/integrations/gRPC-integration.md

```diff
@@ -131,7 +131,7 @@ The following steps will show how to create an app that exposes a server for wit
     	"github.com/golang/protobuf/ptypes/any"
     	"github.com/golang/protobuf/ptypes/empty"
     
-    	commonv1pb "github.com/dapr/dapr/pkg/proto/common/v1"
+    	commonv1pb "github.com/dapr/go-sdk/dapr/proto/common/v1"
```

Comment:
You are right that this needs to change, however the most correct value here would still be `dapr/dapr`. Please change to the following: `commonv1pb "github.com/dapr/dapr/proto/common/v1"`

Reply:
when I checked that location, there is only `proto` file. So, I can't get the stub code for go.

I think we have to choose either `dapr/dapr` or `dapr/go-sdk` to unify the package location of runtime and common.

And, as you already mentioned, If `dapr/dapr` is more correct, we should choose this.

Reply:
@jaeheonji yes, we should use `dapr/dapr` all the way in the native gRPC examples. once you make that change we can merge :)